kernel/aster-nix/src/lib.rs

```diff
@@ -80,6 +80,7 @@ fn init_thread() {
         "[kernel] Spawn init thread, tid = {}",
         current_thread!().tid()
     );
+    aster_frame::enable_interrupts();
```

Comment:
I don't understand. Is this single-line change a bugfix? When are the local IRQs enabled before this change?

Reply:
> I don't understand. Is this single-line change a bugfix?

We must enable interrupts before loading the fs, I think If we use `mount` syscall to do the mounting in the init process. this API can be removed. I will add some comments here to expain it.

> When are the local IRQs enabled before this change?

I searched the code and I found it is enabled in the `execute` function of User Context.

Reply:
> I searched the code and I found it is enabled in the execute function of User Context.

Ok. Then, this is a bugfix. Interrupts should be enabled far earlier. I will accept this line of change as a quick fix. But a permanent solution should be discussed and fixed in a separate PR. Could you add a FIXME comment to this line?

Reply:
Done